backend/models/office_hours/oh_event.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import BaseModel
+from datetime import datetime, date
+from oh_type import OfficeHoursType
+
+__authors__ = ["Sadie Amato"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+class OfficeHoursEvent(BaseModel):
+    """
+    Pydantic model to represent an `OfficeHoursEvent` that has not been created yet.
+
+    This model is based on the `OfficeHoursEventEntity` model, which defines the shape
+    of the `OfficeHoursEvent` database in the PostgreSQL database.
+    """
+    id: int | None = None
+    office_hours_section_id: int
+    location_id: int
+    type: OfficeHoursType
+    title: str
+    description: str
+    location_description: str
```

Comment:
Nit: Curious, would this be extra data added to the room? Wondering if this field is necessary to include in addition to `description`! Also, would this potentially be `| None` if no description is supplied, or will the default be ""?

Reply:
We were thinking this could be optional extra instructions a professor or TA could add to an event. For example, if students should be sitting in a specific area or if there are special instructions for how to get to the location!